packages/project/src/db/project.ts

```diff
@@ -204,25 +205,40 @@ const clone = async (
       ? await db.build.loadByProjectId(project.id, "dev")
       : await db.build.loadByProjectId(project.id, "prod");
 
-  const projectId = uuid();
-  authorizeProject.registerProjectOwner({ projectId }, context);
+  const newProjectId = uuid();
```

Comment:
I think we recently switched to using nanoid everywhere, didn't we @TrySound ?

Reply:
Nanoid for json data and uuid for db rows.

Reply:
why not using one for both? nanoid is apparently even better, no?